config/config.go

```diff
@@ -287,6 +293,10 @@ func (cfg *Config) normallizeTask(taskCfg *TaskConfig) (err error) {
 	} else {
 		taskCfg.PromLabelsBlackList = ""
 	}
+	if taskCfg.Parser == "proto" && cfg.SchemaRegistry.URL == "" {
+		err = errors.Newf("For parser %s schema registry needs", taskCfg.Parser)
```

Comment:
maybe "Schema registry is required for parser %s"?

Reply:
I agree, fixed